src/lib/crypto/krb/prng_fortuna.c

```diff
@@ -443,4 +443,36 @@ krb5_c_random_make_octets(krb5_context context, krb5_data *outdata)
     return 0;
 }
 
+#ifdef _WIN32
+#define BUFLEN 1024
+#else
+#define BUFLEN 64
+#endif
```

Comment:
I see how this preserves the old behavior, but I don't like it.  I will talk through the changes with other team members tomorrow and see if we want to change this behavior (e.g. using a 64-byte buffer on Windows) or if maybe there's a simpler design.